### DISCUSSION TITLE
[The content of a repository has been deleted]

### DISCUSSION THREAD

**Discussion by SamuTechAI9** [community/community#71363] [Codespaces]
**Hi everyone ,**

- First I have to say thank you for accepting me into this wonderful community.

- Now comes the problem:

First I have to say thank you for accepting me into this wonderful community.
Now comes the problem:
It's been about a week since I've logged into VS Code, which I have connected to github[codespace] since my tablet is of low quality. Now I find that my repository has emptied its content (files.py, .txt, etc.) I can't find any type of record on which this action has been done, after looking at the configuration it seems that everything is in order... Could someone tell me what could have happened? I would love to recover my jobs.... Since as a good newbie, I did not make a backup copy of them...

Thank you very much in advance for the help.

**Comment by queenofcorgis** [comment]
Hey there - welcome, we're so glad you're here! I moved this into our Codespaces category so it'll have better visibility among folks who can best assist! Thanks again for posting in the community.
